### PR TITLE
Renamed functions claim,redeem to unwrap,wrap.

### PR DESCRIPTION
**Pull request by seshanthS**
Updated the function names in OSTPrime.sol.

### CODE REVIEW

File: contracts/gateway/OSTPrime.sol [hunk 1]
```diff
@@ -121,7 +121,7 @@ contract OSTPrime is UtilityToken, OSTPrimeConfig {
      *
      * @return `true` if claim was successfully progressed
      */
-    function claim(
+    function unwrap(
```
Comment (sarvesh-ost) [inline contracts/gateway/OSTPrime.sol]: Function documentation and event name should also be updated with new terminology 'unwrap'.

[the remark this answers is not in the document]
Comment (seshanthS) [reply contracts/gateway/OSTPrime.sol]: changed event and function names to 

`wrap,unwrap `

[the remark this answers is not in the document]
Comment (schemar) [reply contracts/gateway/OSTPrime.sol]: Thank you. Please also update the documentation accordingly in all relevant places.

**Comment by deepesh-kn** [comment]
Thanks for your contribution !!!
This is already addressed in the  PR #533, hence closing this PR.

**Pull request closed without merging**